bmcstored_dump_entry.cpp

```diff
@@ -33,6 +36,14 @@ void Entry::delete_()
 
     // Remove Dump entry D-bus object
     phosphor::dump::Entry::delete_();
+
+    // Log PEL for dump delete/offload
+    log<level::INFO>("Log PEL for dump delete or offload");
+
+    phosphor::dump::createPEL(
+        strDumpFileName, "BMC Dump", id,
+        "xyz.openbmc_project.Logging.Entry.Level.Informational",
+        "xyz.openbmc_project.Dump.Error.Invalidate");
```

Comment:
u can do it before delete that way no need to create strDumpFileName variable, nothing serious if this succeeds and delete fails, delete fail is a core dump

Reply:
Moved before delete